src/core/Akka/Actor/BuiltInActors.cs

```diff
@@ -197,6 +197,18 @@ public static object Unwrap(object message)
             }
             return message;
         }
+
+        internal static bool IsDeadLetterSuppressedAnywhere(object message)
```

Comment:
Function to check, up and down the stack, if a message was marked as `IDeadLetterSuppression` anywhere within